server/platform/services/searchengine/bleveengine/search.go

```diff
@@ -315,7 +315,7 @@ func (b *BleveEngine) IndexChannel(_ request.CTX, channel *model.Channel, userID
 	return nil
 }
 
-func (b *BleveEngine) SearchChannels(teamId, userID, term string, isGuest bool) ([]string, *model.AppError) {
+func (b *BleveEngine) SearchChannels(teamId, userID, term string, isGuest, _ bool) ([]string, *model.AppError) {
```

Comment:
FYI - we are not bothered about the Bleve implementation any more because it's deprecated.